### PR TITLE
Move #daala to Libera.Chat.

### PR DESCRIPTION
**Pull request by tdaede**
I'm not sure what to use for webchat, it seems that Libera.Chat
doesn't run a webchat yet.

### CODE REVIEW

**Comment by barrbrain** [comment]
We could include a link to https://libera.chat/ for more information about the network. Their IRC client guide lists https://kiwiirc.com/ as an available web client.

**Review by lu-zero — APPROVED** [review]
I second what @barrbrain suggested.

**Review by barrbrain — APPROVED** [review]

**Comment by negge** [comment]
There has been no public discussion of moving servers. I think this PR is premature.

**Review by negge — CHANGES_REQUESTED** [review]
There has been no public discussion of moving servers. I think this PR is premature.

**Comment by barrbrain** [comment]
For the record, I approve of the content of this change and agree that merging it should be gated on the outcome of public discussion.

File: README.md [hunk 1]
```diff
@@ -176,5 +176,4 @@ Find a full list in feature-table in [`Cargo.toml`](Cargo.toml)
 Please read our guide to [contributing to rav1e](CONTRIBUTING.md).
 
 ## Getting in Touch
-Come chat with us on the IRC channel #daala on Freenode! If you don't have IRC set
-up you can easily connect from your [web browser](http://webchat.freenode.net/?channels=%23daala).
+Come chat with us on the IRC channel #daala on [Libera.Chat](https://libera.chat/)! You can also use https://kiwiirc.com/ to join with a web browser.
```
Comment (vibhoothiiaanand) [inline README.md:179]:
```suggestion
Come chat with us on the IRC channel #daala on [Libera.Chat](https://libera.chat/)! You can also use https://kiwiirc.com/ to join with a [web browser](https://kiwiirc.com/nextclient/#ircs://irc.libera.chat/#daala).
```

**Review by vibhoothiiaanand — COMMENTED** [review]
If moving is going to happen, we can hyperlink the kiwiric chat link like how we did with freenode,

File: README.md (hunk 1 above)
Comment (barrbrain) [reply README.md:179]: Plain URLs like  https://kiwiirc.com/ are automatically linkified.

**Review by negge — COMMENTED** [review]
Please add instructions for matrix users

File: README.md [hunk 2]
```diff
@@ -176,5 +176,4 @@ Find a full list in feature-table in [`Cargo.toml`](Cargo.toml)
 Please read our guide to [contributing to rav1e](CONTRIBUTING.md).
 
 ## Getting in Touch
-Come chat with us on the IRC channel #daala on Freenode! If you don't have IRC set
-up you can easily connect from your [web browser](http://webchat.freenode.net/?channels=%23daala).
+Come chat with us on the IRC channel #daala on [Libera.Chat](https://libera.chat/)! You can also use [web browser](https://kiwiirc.com/nextclient/#ircs://irc.libera.chat/#daala) to join with a web browser.
```
Comment (negge) [inline README.md:179]: Can we use this link instead?

https://web.libera.chat/?channel=#daala

**Review by negge — CHANGES_REQUESTED** [review]

**Review by tmatth — APPROVED** [review]

**Review by negge — APPROVED** [review]
LGTM

**Comment by vibhoothi** [comment]
@tdaede can we land this

File: README.md [hunk 3]
```diff
@@ -178,5 +178,4 @@ Find a full list in feature-table in [`Cargo.toml`](Cargo.toml)
 Please read our guide to [contributing to rav1e](CONTRIBUTING.md).
 
 ## Getting in Touch
-Come chat with us on the IRC channel #daala on Freenode! If you don't have IRC set
-up you can easily connect from your [web browser](http://webchat.freenode.net/?channels=%23daala).
+Come chat with us on the IRC channel #daala on [Libera.Chat](https://libera.chat/)! You can also use [web browser](https://web.libera.chat/?channel=#daala) to join with a web browser.
```
Comment (barrbrain) [inline README.md:181]:
```suggestion
Come chat with us on the IRC channel #daala on [Libera.Chat](https://libera.chat/)! You can also use a [web client](https://web.libera.chat/?channel=#daala) to join with a web browser.
```

**Pull request merged**